src/core/zowe/core_for_zowe_sdk/profile_manager.py

```diff
@@ -169,12 +171,57 @@ def get_profile(
             )
         finally:
             return cfg_profile
+    
+    @staticmethod
+    def load_schema(
+        cfg: ConfigFile,
+        path_config_json: str,
+        opt_in: Optional[bool] = True,
+    ) -> str:
+        """
+        Get the $schema_property from the config and load the schema
+
+        Returns
+        -------
+        file_path to the $schema property
+        """
+
+        path_schema_json = None
+        try:
+            path_schema_json = cfg.load_schema()
+            if path_schema_json is None:    # check if the $schema property is not defined
+                warnings.warn(
+                    f"$schema property could not found"
+                )
+                
+            # validate the $schema property 
+            if path_schema_json and opt_in:
+                validate_config_json(path_config_json, path_schema_json)
```

Comment:
Can we enhance the `validate_config_json` method to download the schema from the internet if `path_schema_json` points to an Internet URI rather than local file path?